lex.dd

```diff
@@ -746,24 +735,23 @@ $(GNAME HexLetter):
     $(B D)
     $(B E)
     $(B F)
-    $(D _)
 )
 
-	$(P Integers can be specified in decimal, binary, octal, or hexadecimal.)
+	$(P Integers can be specified in decimal, binary or hexadecimal.)
```

Comment:
I deleted the comma accidentally.

I'm new to this ... I made this edit from the "Improve this page" button at http://dlang.org/lex.html
Can someone add the comma for me, or do I need to resubmit this, or ... ?